gulpfile.cjs

```diff
@@ -423,6 +437,16 @@ gulp.task(
     //See https://github.com/CesiumGS/cesium/pull/3106#discussion_r42793558 for discussion.
     glslToJavaScript(false, "Build/minifyShaders.state");
 
+    // Remove prepare step from package.json to avoid redownloading Draco3d files
+    const packageJsonData = fs.readFileSync("./package.json");
+    const buildPackageJson = JSON.parse(packageJsonData);
+    const scripts = buildPackageJson.scripts;
+    delete scripts.prepare;
+    fs.writeFileSync(
```

Comment:
This will overwrite the package.json at the root of the repo.. is that fine here? Should this somehow feed directly into `mergeStream` without the call to `writeFileSync`?

Reply:
Also I'm seeing an Access Denied error here in the CI logs.

Reply:
> This will overwrite the package.json at the root of the repo.. is that fine here? Should this somehow feed directly into `mergeStream` without the call to `writeFileSync`?

Ideally we shouldn't have to write to disk first.

But worst case, overwrite `package.json` temporarily but overwrite it back to its original form after the the zip file is complete.